package.json

```diff
@@ -82,7 +84,8 @@
     "remap-istanbul": "0.10.1",
     "rimraf": "2.5.4",
     "rollup": "0.55.3",
-    "rxjs": "^5.4.2",
+    "rxjs": "^6.1.0",
+    "rxjs-compat": "^6.1.0",
```

Comment:
Can we remove this dependency since you updated the the code to use the new RxJS 6.X.X format?